packages/opentelemetry-api/src/trace/NoopSpan.ts

```diff
@@ -59,11 +59,6 @@ export class NoopSpan implements Span {
     return this;
   }
 
-  // By default does nothing
-  addLink(spanContext: SpanContext, attributes?: Attributes): this {
```

Comment:
what's the reason of removing that ?

Reply:
`NoopSpan` implements `interface Span` but `addLink` was removed there long time ago. It looks like it was forgotten to remove it here also.
If needed I can move this change in a separate PR.

Reply:
Good catch